cmd/server/assets/static/js/application.js

```diff
@@ -275,3 +274,38 @@ function getCookie(cname) {
   }
   return "";
 }
+
+// Common error codes which should cancel the whole upload.
+let stopUploadingCodes = [
```

Comment:
It'd be a larger refactor, but should we get these from the server somehow?

Reply:
That's a good idea. I also want to include the detailed error const eg "maintenance_mode" rather than just HTTP code. I'll save that all for another PR and do them together